drivers/hmis/app/graphql/types/hmis_schema/client_alert.rb

```diff
@@ -0,0 +1,19 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::ClientAlert < Types::BaseObject
+    description 'Alert'
+    field :id, ID, null: false
+    field :note, String, null: false
+    field :created_at, GraphQL::Types::ISO8601DateTime, null: false
+    field :created_by, Types::Application::User, null: true
+    field :expiration_date, GraphQL::Types::ISO8601DateTime, null: true
+    field :severity, String, null: true
```

Comment:
I think we landed on `priority` for the user-facing language, so lets use that in the api schema too.

Also - can you create a GraphQL enum for this?

`Types::HmisSchema::Enums::ReminderTopic` is one example to look at. It reads the possible values from a different class. You could do the same thing, reading the priority levels from the ClientAlert model: `Hmis::ClientAlert::PRIORITY_LEVELS`. Where the values are`['low','medium','high']`. You can also add a validation rule on ClientAlert, which will ensure that any value written to the severity field is one of the approved strings. (See rails `validates` docs with  `inclusion`)

Reply:
And lets make it `null: false`. You can add `default_value: 'low'`

Reply:
@gigxz this all sounds good to me, but I'm not quite sure how to create a module `Hmis::ClientAlert` with `PRIORITY_LEVELS` inside it when `Hmis::ClientAlert` already exists as a class? Is that what you're indicating here, or should it be named something else in order to not conflict?

Reply:
`PRIORITY_LEVELS` would be a constant thats defined inside the existing class

Reply:
of course haha 🤦‍♀️